HardwareCheckout/config.py

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env python3
-import os
-db_path = 'sqlite://' + os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../database/db.sqlite')
+#import os
+#db_path = 'sqlite://' + os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../database/db.sqlite')
+xpath = "/opt/database/db.sqlite"
```

Comment:
This will work for now, but I would like to see this be more dynamic so that we are not relying on hardcoded paths.